### ISSUE TITLE
AppLovinSdk: [DataCollector] Could not collect Google Advertising ID

### ISSUE THREAD

**Issue by jesphinpt** [AppLovin/AppLovin-MAX-SDK-Android#560]
### MAX SDK Version

11.11.3

### Device/Platform Info

Android

### Current Behavior

Getting the error Could not collect Google Advertising ID while running the app. 

I have 

 api 'com.google.android.gms:play-services-ads:22.4.0'
  api 'com.google.android.gms:play-services-analytics:18.0.3'
    api 'com.google.android.gms:play-services-auth:20.7.0'
   implementation 'com.google.android.gms:play-services-location:21.0.1'

### How to Reproduce

Add the applovin sdk and try to intialize and run the app I got the  error.

**Comment by thomasmso** [comment]
@jesphinpt - Could you send the logs containing the error?

**Comment by jesphinpt** [comment]
I am getting two different issues



1.Could not collect Google Advertising ID - 

```
E/AppLovinSdk: [DataCollector] Could not collect Google Advertising ID - this will negatively impact your eCPMs! Please integrate the Google Play Services SDK into your application. More info can be found online at http://developer.android.com/google/play-services/setup.html. If you're sure you've integrated the SDK and are still seeing this message, you may need to add a ProGuard exception: -keep public class com.google.android.gms.** { public protected *; }
    java.io.IOException: java.util.concurrent.TimeoutException: Timed out waiting for the service connection
        at com.google.android.gms.ads.identifier.AdvertisingIdClient.zzb(com.google.android.gms:play-services-ads-identifier@@17.1.0:16)
        at com.google.android.gms.ads.identifier.AdvertisingIdClient.getAdvertisingIdInfo(com.google.android.gms:play-services-ads-identifier@@17.1.0:3)
        at com.applovin.impl.sdk.utils.d.collectGoogleAdvertisingInfo(SourceFile:173)
        at com.applovin.impl.sdk.utils.d.c(SourceFile:142)
        at com.applovin.impl.sdk.utils.d.b(SourceFile:88)
        at com.applovin.impl.sdk.utils.d.a(SourceFile:52)
        at com.applovin.impl.sdk.utils.j$1.run(SourceFile:37)
        at java.lang.Thread.run(Thread.java:764)
```

2. AppLovinSdk: [DataCollector] Unable to determine if Fire OS limited ad tracking is turned o



```
E/AppLovinSdk: [DataCollector] Unable to determine if Fire OS limited ad tracking is turned on
    android.provider.Settings$SettingNotFoundException: limit_ad_tracking
        at android.provider.Settings$Secure.getIntForUser(Settings.java:4972)
        at android.provider.Settings$Secure.getInt(Settings.java:4958)
        at com.applovin.impl.sdk.utils.d.collectFireOSAdvertisingInfo(SourceFile:219)
        at com.applovin.impl.sdk.utils.d.c(SourceFile:147)
        at com.applovin.impl.sdk.utils.d.b(SourceFile:88)
        at com.applovin.impl.sdk.utils.d.a(SourceFile:52)
        at com.applovin.impl.sdk.utils.j$1.run(SourceFile:37)
        at java.lang.Thread.run(Thread.java:764)
```

**Comment by Devik0213** [comment]
Hello, @thomasmso 

I am an Android developer named Harry from Korea who uses AppLovin.

I am also using Applovin `11.11.3` and experiencing the same error. It seems to occur when the Google Play version is low, (or on Emulator without Playstore) and I came to check if Applovin has a version that solves this issue, but there is no specific mention in the Release Note, so I am leaving a comment while checking the issues.

**I wonder if there is a version that has been resolved.**

I will wait for your reply. Thank you.
